packages/sync/src/Actions.php

```diff
@@ -132,34 +182,72 @@ static function sync_allowed() {
 		return true;
 	}
 
-	static function sync_via_cron_allowed() {
+	/**
+	 * Determines if syncing during a cron job is allowed.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @return bool
```

Comment:
Default value in `Defaults` seem to be `1`, can this be an `int` too?

Reply:
I guess??